nifty-ssl/src/main/java/com/facebook/nifty/ssl/OpenSslServerConfiguration.java

```diff
@@ -62,6 +62,9 @@
         public long sessionTimeoutSeconds = 86400;
         public long sessionCacheSize = 0;
         public boolean enableStatefulSessionCache = true;
+        public int maxSslBufferSize = 19267584;
```

Comment:
I think this might be a bit too large as a default for the thread local buffer. 

this is about 18mb. If we have about 80 threads which is practical, we'll end up using a gig for these buffers.